File: Python/URI/Beginner/1094.py
n = input()
n = int(n)

total = 0
_totalC = 0
_totalR = 0
_totalS = 0

for i in range(n):
	x, y = input().split()
	x = int(x)
	total += x

	if (y == 'C'):
		_totalC += x
	elif (y == 'R'):
		_totalR += x
	else:
		_totalS += x

a = (_totalC/total)*100
b = (_totalR/total)*100
c = (_totalS/total)*100


print('Total: %d cobaias' % total)
print('Total de coelhos: %d' % _totalC)
print('Total de ratos: %d' % _totalR)
print('Total de sapos: %d' % _totalS)
print('Percentual de coelhos: %.2f %%' % a)
print('Percentual de ratos: %.2f %%' % b)
print('Percentual de sapos: %.2f %%' % c)

File: URI/Beginner/1094.cpp

// g++ -std=c++11 file.cpp (compile c++11)

#include <iostream>
#include <vector>
#include <algorithm>
#include <string>
#include <sstream> 
#include <limits.h> // INT_MIN AND INT_MAX
#include <bitset>
#include <cmath>

#define ll long long

using namespace std;

int main() {

    int testCases, amount, sumCoelho=0, sumRato=0, sumSapo=0;
    int globalSum = 0;
    char animal;

    scanf("%d", &testCases);

    while (testCases--) {
        cin >> amount >> animal;

        if (animal == 'C') {
            sumCoelho += amount;
        } else if (animal == 'R') {
            sumRato += amount;
        } else {
            sumSapo += amount;
        }

        globalSum += amount;
    }


    printf("Total: %d cobaias\n", globalSum);

    printf("Total de coelhos: %d\n", sumCoelho );
    printf("Total de ratos: %d\n", sumRato);
    printf("Total de sapos: %d\n", sumSapo);

    printf("Percentual de coelhos: %.2lf %%\n", (sumCoelho*100.0)/globalSum);
    printf("Percentual de ratos: %.2lf %%\n", (sumRato*100.0)/globalSum);
    printf("Percentual de sapos: %.2lf %%\n", (sumSapo*100.0)/globalSum);

    
    
    return 0;
}